split.Console/Interfaces/IGameInterface.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a very rough draft. It will eventually be implemented by a Game
+    /// controller. This interface encapsulates game-centered actions. 
+    /// </summary>
+    public interface IGameInterface
+    {
+        /// <summary>
+        /// This will launch the Game object containing all necessary information
+        /// about the game. It takes a game name as an argument. This will initialize the game. 
+        /// The API call from the front end could be /game/{name}/launch? 
+        /// </summary>
+        /// <returns>An object that implements IGameInterface. </returns>
+        IGameInterface launch(string name);
```

Comment:
The assumption here is that a player to starts the game gets to select what kind of game he is starting through menu options. This is where the name of the game will come from.